test/public/runs/overview.test.js

```diff
@@ -656,25 +665,29 @@ module.exports = () => {
 
         // Open filter toggle
         await pressElement(page, '#openFilterToggle');
-        await waitForTimeout(200);
+        await page.waitForSelector(offFilterSelector);
 
         await page.$eval(offFilterSelector, (element) => element.click());
-        await waitForTimeout(300);
-        table = await page.$$('tbody tr');
+        await waitForTableLength(page, 9);
 
-        expect(table.length).to.equal(8);
+        table = await page.$$('tbody tr');
+        expect(table.length).to.equal(9);
```

Comment:
maybe waitForTableLength?

Reply:
Already waiting for table length above, so I removed this. But is it really a good idea to use waitForTableLength instead of an assertion? This way if it fails you won't know the actual value right away